examples/Workflows/nmt_tranformers_pipeline/nmt_workflow_dualtranslation.yaml

```diff
@@ -4,7 +4,7 @@ models:
     batch-size: 4
     max-batch-delay: 100
     retry-attempts: 5
-    timeout-ms: 10000
+    timeout-ms: 300000
```

Comment:
@maaquib Is this timeout only sepecific to the nmt workflow test or will apply for the entire Postman collection?

Reply:
@chauhang Just to the nmt and cv workflows. I increased these since it takes longer to download these mar files from s3